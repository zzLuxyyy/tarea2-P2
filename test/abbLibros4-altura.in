crearABBLibrosVacio
alturaABBLibros
liberarABBLibros

crearABBLibrosVacio
crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
insertarLibroABBLibros
alturaABBLibros
liberarABBLibros

crearABBLibrosVacio
crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros
crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
insertarLibroABBLibros
alturaABBLibros

crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
insertarLibroABBLibros
alturaABBLibros

crearFecha 01/01/2002
crearLibro 849550144 "Juventud" "J.M." "Coetzee" "Un joven sudafricano busca sentido en su vida y vocación en Londres" 4
insertarLibroABBLibros
alturaABBLibros

crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros
alturaABBLibros

crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
insertarLibroABBLibros
alturaABBLibros

liberarABBLibros
Fin
