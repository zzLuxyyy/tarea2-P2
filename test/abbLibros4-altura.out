1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>La altura del arbol es 0.
3>4>Se ejecutó crearTABBLibrosVacio exitosamente.
5>6>7>Se insertó el libro con ISBN 999001122 al árbol.
8>La altura del arbol es 1.
9>10>Se ejecutó crearTABBLibrosVacio exitosamente.
11>12>13>Se insertó el libro con ISBN 316769177 al árbol.
14>15>16>Se insertó el libro con ISBN 140003446 al árbol.
17>La altura del arbol es 2.
18>19>20>Se insertó el libro con ISBN 204578912 al árbol.
21>La altura del arbol es 3.
22>23>24>Se insertó el libro con ISBN 849550144 al árbol.
25>La altura del arbol es 3.
26>27>28>Se insertó el libro con ISBN 843397030 al árbol.
29>La altura del arbol es 3.
30>31>32>Se insertó el libro con ISBN 880619315 al árbol.
33>La altura del arbol es 3.
34>35>Fin.
